src/components/ListItem.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import Checkbox from '../components/Checkbox';
 
-const ListItem = ({ item, shoppingList }) => (
-  <li className="list__item">
+const ListItem = ({ index, item, shoppingList }) => (
+  <li className={index} aria-label={index}>
```

Comment:
The aria-label attribute isn't supported by non-interactive elements like <li>. It's better if you add it on the label of the checkbox :slightly_smiling_face: . Also, it's better if the aria is something human readable, like "Kind of soon", instead of "kind-of-soon" :+1:

Reply:
@faysvas thank you for pointing this out, I've made the changes. In the latest commit I:
- Created a new variable for the aria-label, and used human-readable wording for the indices.
  I created a new variable because removing the hyphens would give issues with passing the prop as a className.
- Added the aria-label to a parent div instead of the label because adding it to the label meant the screen-reader would then ignore the name of the item.

The changes to the legend are basically to make it more accessible.